src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.util.Scanner;
+
 public class Duke {
+    private static final String EVENT = "event";
+    private static final String TODO = "todo";
+    private static final String DEADLINE = "deadline";
+
+    private static final String SEPARATOR = "____________________________________________________________";
+    private static final int LENGTH = 100;
```

Comment:
It is unclear if this constant indicates task length of a single task or maximum number of tasks, could be renamed for better readability.

Code Quality Guideline: Name well